js/chatbot/src/ChatBot.svelte

```diff
@@ -39,7 +40,7 @@
 		gfm: true,
 		breaks: true,
 		pedantic: false,
-		sanitize: true,
+		sanitize: false,
 		smartLists: true,
 		smartypants: false
```

Comment:
We are getting some warnings about deprecations (which is slightly annoying because they are defaults and not being set by us) but lets explicitly set them to false here:

```suggestion
		smartypants: false,
		headerIds: false,
		mangle: false
```

Reply:
We are also getting a warning about the highlight method, so lets fix that here too. Instead of using `marked.setOptions({ highlight: ...})`. We need to add `marked-highlight` as a dependency of chabot and then do something like the following:

```js
	import { markedHighlight } from "marked-highlight";
  ...
  
  marked.use(
		markedHighlight({
			highlight(code, lang) {
				return Prism.highlight(code, Prism.languages[lang], lang);
			}
		})
	);
```

I tested this and it seems to work fine.

Reply:
Actually I need to make some changes to the other chatbot PR. I'll do this there and we can merge them in here.